### PR TITLE
added name on client

### PR DESCRIPTION
**Pull request by tcitworld**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | yes
| Fixed tickets | #1765
| License       | MIT

Needs migrations.

Following #2107

### CODE REVIEW

File: src/Wallabag/ApiBundle/Entity/Client.php [hunk 1]
```diff
@@ -18,8 +18,39 @@ class Client extends BaseClient
      */
     protected $id;
 
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="title", type="text", nullable=true)
```
Comment (j0k3r) [inline src/Wallabag/ApiBundle/Entity/Client.php]: Shoudn't this be `name` instead?

**Comment by nicosomb** [comment]
Maybe we can add the userId (for the client) in this PR?

**Comment by j0k3r** [comment]
@nicosomb does your comment is related to that issue https://github.com/wallabag/wallabag/issues/2062 ?

**Comment by j0k3r** [comment]
And we should close this PR and re-open in to target the 2.1 branch

**Comment by nicosomb** [comment]
@j0k3r yes it was about this bug

**Comment by tcitworld** [comment]
Closed in favour of #2171

**Pull request closed without merging**